lib/compat/wordpress-6.6/class-wp-rest-media-search-handler-gutenberg-6-6.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * REST API: WP_REST_Media_Search_Handler class
+ *
+ * @package WordPress
+ * @subpackage REST_API
```

Comment:
I replicated the intention of [this PR](https://github.com/WordPress/gutenberg/pull/58638) and gave credit to the author in case this one gets merged first